sdk/ai/azure-ai-resources/setup.py

```diff
@@ -66,9 +66,9 @@
     python_requires="<4.0,>=3.8",
     install_requires=[
         # NOTE: To avoid breaking changes in a major version bump, all dependencies should pin an upper bound if possible.
-        "azure-ai-ml",
-        "mlflow<3",
-        "azure-mgmt-authorization"
+        "azure-ai-ml~=1.0",
+        "mlflow-skinny<3",
+        "azure-mgmt-authorization>=0.40.0"
```

Comment:
remove `mgmt-authorization` here too

Reply:
Resolved by https://github.com/Azure/azure-sdk-for-python/pull/32770/commits/3e7739f2afab4ef36e3c915950bf4309676954d9